### PR TITLE
[Docs] Add Chinese version of model_zoo

### PR DESCRIPTION
**Pull request by jbwang1997**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs_zh-CN/model_zoo.md [hunk 1]
```diff
@@ -1,3 +1,319 @@
 # 模型库
 
 ## 镜像地址
+
+从MMDetection V2.0起，我们只通过阿里云维护模型库。V1.x版本的模型依然保存在亚马逊并且将会在未来逐渐弃用。
```
Comment (ZwwWayne) [inline docs_zh-CN/model_zoo.md:5]: 'V1.x版本的模型已经弃用。'Can also update the English version if you have time

File: docs_zh-CN/model_zoo.md [hunk 2]
```diff
@@ -1,3 +1,319 @@
...
+
+- 所有模型都是在`coco_2017_train`上训练，在`coco_2017_val`上测试。
+- 我们使用分布式训练。
+- 所有pytorch-style的ImageNet预训练主干网络来自PyTorch的模型库，caffe-style的预训练主干网络来自detectron2最新开源的模型。
+- 为了与其他代码库公平比较，文档中所写的GPU内存是8个GPU的`torch.cuda.max_memory_allocated()`的最大值，此值通常小于 nvidia-smi 显示的值。 
+- 我们所写的模型推理时间包含网络前向时间和后处理时间，不包含数据加载时间。所有结果通过[benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/benchmark.py)脚本计算所得。该脚本会计算推理2000张图像的平均时间。
+
+## ImageNet预训练模型
+
+通过ImageNet分类任务预训练的主干网络进行初始化是很常见的操作。所有预训练模型的链接都可以在[open_mmlab](https://github.com/open-mmlab/mmcv/blob/master/mmcv/model_zoo/open_mmlab.json)中找到。根据`img_norm_cfg`和出处，我们可以将所有 ImageNet 预训练模型分为几种情况：
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:17]: 1. space
2. `img_norm_cfg` 和出处，出处有点奇怪，英文是 source of weight，翻译成 `原始权重` 可能会好一点，或者有更好的可以替换

File: docs_zh-CN/model_zoo.md [hunk 3]
```diff
@@ -1,3 +1,319 @@
...
+- MSRA styles：[MSRA](https://github.com/KaimingHe/deep-residual-networks) 模型权重，包含ResNet50_Caffe，ResNet101_Caffe。`img_norm_cfg`为`dict(
+    mean=[103.530, 116.280, 123.675], std=[1.0, 1.0, 1.0], to_rgb=False)`。
+- Caffe2 styles：现阶段只包含ResNext101_32x8d。`img_norm_cfg`为`dict(mean=[103.530, 116.280, 123.675], std=[57.375, 57.120, 58.395], to_rgb=False)`。
+- Other styles: SSD的`img_norm_cfg`为`dict(mean=[123.675, 116.28, 103.53], std=[1, 1, 1], to_rgb=True)`，YOLOv3的`img_norm_cfg`为`dict(mean=[0, 0, 0], std=[255., 255., 255.], to_rgb=True)`。
+
+MMdetection常用到的主干网络细节如下表所示：
+
+| 模型             | 来源         | 链接                                                         | 描述                                                  |
+| ---------------- | ----------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| ResNet50         | TorchVision | [torchvision's ResNet-50](https://download.pytorch.org/models/resnet50-19c8e357.pth) | 来自[torchvision's ResNet-50](https://download.pytorch.org/models/resnet50-19c8e357.pth)。 |
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:31]: `torchvision's ResNet-50` 中的 `'s` 去掉，翻译成 `torchvision 中的 ResNet-50` 可能会好一点

Link 中的可以不改，但是 描述中的最好改一下。，这个表格需要整体更新

File: docs_zh-CN/model_zoo.md [hunk 4]
```diff
@@ -1,3 +1,319 @@
...
+| ResNet50_Caffe   | MSRA        | [MSRA's ResNet-50](https://download.openmmlab.com/pretrain/third_party/resnet50_caffe-788b5fa3.pth) | 由[Detectron2's R-50.pkl](https://dl.fbaipublicfiles.com/detectron2/ImageNetPretrained/MSRA/R-50.pkl)转化的副本。原始权重文件来自[MSRA's original ResNet-50](https://github.com/KaimingHe/deep-residual-networks)。|
+| ResNet101_Caffe  | MSRA        | [MSRA's ResNet-101](https://download.openmmlab.com/pretrain/third_party/resnet101_caffe-3ad79236.pth) | 由[Detectron2's R-101.pkl](https://dl.fbaipublicfiles.com/detectron2/ImageNetPretrained/MSRA/R-101.pkl)转化的副本。原始权重文件来自[MSRA's original ResNet-101](https://github.com/KaimingHe/deep-residual-networks)。|
+| ResNext101_32x8d | Caffe2      | [Caffe2 ResNext101_32x8d](https://download.openmmlab.com/pretrain/third_party/resnext101_32x8d-1516f1aa.pth) | 由[Detectron2's X-101-32x8d.pkl](https://dl.fbaipublicfiles.com/detectron2/ImageNetPretrained/FAIR/X-101-32x8d.pkl)转化的副本。原始ResNeXt-101-32x8d由FB使用Caffe2训练。|
+
+## Baselines
+
+### RPN
+
+请参考[RPN](https://github.com/open-mmlab/mmdetection/blob/master/configs/rpn)。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:42]: space

File: docs_zh-CN/model_zoo.md [hunk 5]
```diff
@@ -1,3 +1,319 @@
...
+
+请参考[CenterNet](https://github.com/open-mmlab/mmdetection/blob/master/configs/centernet)。
+
+### Other datasets
+
+我们还在[PASCAL VOC](https://github.com/open-mmlab/mmdetection/blob/master/configs/pascal_voc)，[Cityscapes](https://github.com/open-mmlab/mmdetection/blob/master/configs/cityscapes)和[WIDER FACE](https://github.com/open-mmlab/mmdetection/blob/master/configs/wider_face)上对一些方法进行了基准测试。
+
+### Pre-trained Models
+
+我们还通过多尺度训练、更长的训练策略来训练用ResNet-50和[RegNetX-3.2G](https://github.com/open-mmlab/mmdetection/blob/master/configs/regnet)作为主干网络的[Faster R-CNN](https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn)和[Mask R-CNN](https://github.com/open-mmlab/mmdetection/blob/master/configs/mask_rcnn)。这些模型可以作为下游任务的预训练模型。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:234]: 多尺度训练和更长的训练策略

File: docs_zh-CN/model_zoo.md [hunk 6]
```diff
@@ -1,3 +1,319 @@
...
+
+## 速度基准
+
+### 训练速度基准
+
+我们提供[analyze_logs.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/analyze_logs.py)来得到训练中每一次迭代的平均时间。示例请参考[Log Analysis](https://mmdetection.readthedocs.io/en/latest/useful_tools.html#log-analysis)。
+
+我们与其他流行框架的Mask R-CNN训练速度进行比较（数据是从[detectron2](https://github.com/facebookresearch/detectron2/blob/master/docs/notes/benchmarks.md/)复制而来）。mmdetection上，我们使用[mask_rcnn_r50_caffe_fpn_poly_1x_coco_v1.py](https://github.com/open-mmlab/mmdetection/blob/master/configs/mask_rcnn/mask_rcnn_r50_caffe_fpn_poly_1x_coco_v1.py)进行基准测试。它与detectron2的[mask_rcnn_R_50_FPN_noaug_1x.yaml](https://github.com/facebookresearch/detectron2/blob/master/configs/Detectron1-Comparisons/mask_rcnn_R_50_FPN_noaug_1x.yaml)设置完全一样。同时，我们还提供了[模型权重](https://download.openmmlab.com/mmdetection/v2.0/benchmark/mask_rcnn_r50_caffe_fpn_poly_1x_coco_no_aug/mask_rcnn_r50_caffe_fpn_poly_1x_coco_no_aug_compare_20200518-10127928.pth)和[训练log](https://download.openmmlab.com/mmdetection/v2.0/benchmark/mask_rcnn_r50_caffe_fpn_poly_1x_coco_no_aug/mask_rcnn_r50_caffe_fpn_poly_1x_coco_no_aug_20200518_105755.log.json)作为参考。为了跳过GPU预热时间，吞吐量按照100-500次迭代之间的平均吞吐量来计算。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:242]: 在 mmdetection 中，

File: docs_zh-CN/model_zoo.md [hunk 7]
```diff
@@ -1,3 +1,319 @@
...
+| [MMDetection](https://github.com/open-mmlab/mmdetection)                               | 61                 |
+| [maskrcnn-benchmark](https://github.com/facebookresearch/maskrcnn-benchmark/)          | 53                 |
+| [tensorpack](https://github.com/tensorpack/tensorpack/tree/master/examples/FasterRCNN) | 50                 |
+| [simpledet](https://github.com/TuSimple/simpledet/)                                    | 39                 |
+| [Detectron](https://github.com/facebookresearch/Detectron)                             | 19                 |
+| [matterport/Mask_RCNN](https://github.com/matterport/Mask_RCNN/)                       | 14                 |
+
+### 推理时间基准
+
+我们提供[benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/benchmark.py)对推理时间进行基准测试。此脚本将推理2000张图片并计算忽略前5次推理的平均推理时间。可以通过`LOG-INTERVAL`来改变log输出间隔（默认50）。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:256]: 通过设置 `LOG-INTERVAL`

File: docs_zh-CN/model_zoo.md [hunk 8]
```diff
@@ -1,3 +1,319 @@
...
+
+- Python 3.7
+- PyTorch 1.4
+- CUDA 10.1
+- CUDNN 7.6.03
+- NCCL 2.4.08
+
+### 精度
+
+| Type                                                                                                                                   | Lr schd | Detectron2                                                                                                                             | mmdetection | Download                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:284]: type --> 模型
Lr schd --> 训练策略
Downloas --> 下载

File: docs_zh-CN/model_zoo.md [hunk 9]
```diff
@@ -1,3 +1,319 @@
...
+| Mask R-CNN   | 22.5 (23.3) | 19.6        |
+| Retinanet    | 17.8 (18.2) | 20.6        |
+
+### 训练内存
+
+| Type         | Detectron2 | mmdetection |
+| ------------ | ---------- | ----------- |
+| Faster R-CNN | 3.0        | 3.8         |
+| Mask R-CNN   | 3.4        | 3.9         |
+| Retinanet    | 3.9        | 3.4         |
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:319]: 请确保：中文 English 中文 都更改了

File: docs_zh-CN/model_zoo.md [hunk 10]
```diff
@@ -1,3 +1,318 @@
...
+
+从 MMDetection V2.0 起，我们只通过阿里云维护模型库。V1.x 版本的模型已经弃用。
+
+## 共同设置
+
+- 所有模型都是在 `coco_2017_train` 上训练，在 `coco_2017_val` 上测试。
+- 我们使用分布式训练。
+- 所有 pytorch-style 的 ImageNet 预训练主干网络来自 PyTorch 的模型库，caffe-style 的预训练主干网络来自 detectron2 最新开源的模型。
+- 为了与其他代码库公平比较，文档中所写的 GPU 内存是8个 GPU 的 `torch.cuda.max_memory_allocated()` 的最大值，此值通常小于 nvidia-smi 显示的值。 
+- 我们以网络foward和后处理的时间加和作为推理时间，不包含数据加载时间。所有结果通过[benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/benchmark.py)脚本计算所得。该脚本会计算推理2000张图像的平均时间。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:13]: 1. 网络 foward 和...
2. 中文 数字 中文，also need space  （推理 2000 张图像）

File: docs_zh-CN/model_zoo.md [hunk 11]
```diff
@@ -1,3 +1,318 @@
...
+| [MMDetection](https://github.com/open-mmlab/mmdetection)                               | 61                 |
+| [maskrcnn-benchmark](https://github.com/facebookresearch/maskrcnn-benchmark/)          | 53                 |
+| [tensorpack](https://github.com/tensorpack/tensorpack/tree/master/examples/FasterRCNN) | 50                 |
+| [simpledet](https://github.com/TuSimple/simpledet/)                                    | 39                 |
+| [Detectron](https://github.com/facebookresearch/Detectron)                             | 19                 |
+| [matterport/Mask_RCNN](https://github.com/matterport/Mask_RCNN/)                       | 14                 |
+
+### 推理时间基准
+
+我们提供 [benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/benchmark.py) 对推理时间进行基准测试。此脚本将推理2000张图片并计算忽略前5次推理的平均推理时间。可以通过设置 `LOG-INTERVAL` 来改变 log 输出间隔（默认50）。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:255]: 1. space 2000 space
2. 前 5 次
3. 默认 50

**Review by BIGWangYuDong — COMMENTED** [review]
LGTM

File: docs_zh-CN/model_zoo.md [hunk 12]
```diff
@@ -252,7 +252,7 @@ MMdetection 常用到的主干网络细节如下表所示：
 
 ### 推理时间基准
 
-我们提供 [benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/benchmark.py) 对推理时间进行基准测试。此脚本将推理2000张图片并计算忽略前5次推理的平均推理时间。可以通过设置 `LOG-INTERVAL` 来改变 log 输出间隔（默认50）。
+我们提供 [benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/benchmark.py) 对推理时间进行基准测试。此脚本将推理 2000 张图片并计算忽略前 5 次推理的平均推理时间。可以通过设置 `LOG-INTERVAL` 来改变 log 输出间隔（默认 50）。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/model_zoo.md:255]: 默认为 50

**Review by BIGWangYuDong — COMMENTED** [review]
LGTM

**Comment by BIGWangYuDong** [comment]
Ping @AronLin

**Review by ZwwWayne — APPROVED** [review]

**Comment by ZwwWayne** [comment]
Please fix the lint issues before the merge.

**Review by BIGWangYuDong — APPROVED** [review]

**Pull request merged**